python_modules/dagster/dagster/_daemon/asset_daemon.py

```diff
@@ -1161,6 +1173,27 @@ def submit_run_request(
                     )
                     updated_evaluation_keys.add(entity_key)
 
+            # check if the sensor is still enabled:
+            check_after_runs_num = instance.get_tick_termination_check_interval()
```

Comment:
I moved the check for the check interval here so that we can support a sensor being turned off and then the setting changing, while we're in the intermediate time before the UI changes land. Are there issues with fetching a setting this frequently?